src/main/kotlin/org/taktik/icure/be/ehealth/logic/kmehr/incapacity/impl/v20170601/IncapacityExport.kt

```diff
@@ -355,6 +360,12 @@ class IncapacityExport(
 									contents.add(
 										ContentType().apply {
 											person = PersonType().apply {
+												//NOTE: Mult-eMediatt only requires the phone number of the contactperson and no other data.
+												//		But kmehr requires name/id/sex so dummy values are given here
+												ids.add(IDPATIENT().apply { s = IDPATIENTschemes.LOCAL; sv = config.soft?.version; sl = "${config.soft?.name}-Person-Id"; value = "cp" })
+												firstnames.add("contactpersontel")
+												familyname = "contactpersontel"
```

Comment:
contactpersontel ?

Reply:
For mult-emediatt we only have the telephone number, but kmehr persontype requires names/id/sex so we provide dummy values here, it's uggly but it's what ehealth wants. (do you want me to put the more neutral 'dummyName' ?)